src/main/sources/GA4GHAlignmentSource.js

```diff
@@ -52,6 +41,9 @@ function create(spec: GA4GHSpec): AlignmentDataSource {
   var coveredRanges: ContigInterval<string>[] = [];
 
   function addReadsFromResponse(response: Object) {
+    if (response.alignments === undefined) {
+      return;
```

Comment:
This occurs when you query a region without data. For example, for variants, if you query the GA4GH server:
`curl --data '{"variantSetId": "WyIxa2dlbm9tZXMiLCJ2cyIsInBoYXNlMy1yZWxlYXNlIl0", "end": "10200", "pageSize": "0", "pageToken": "", "start": "10000", "callSetIds": ["WyIxa2dlbm9tZXMiLCJ2cyIsInBoYXNlMy1yZWxlYXNlIiwiSEcwMDA5NiJd"],  "referenceName": "10"}' --header 'Content-Type: application/json'     http://1kgenomes.ga4gh.org/variants/search`

It will return `{}`

Therefore, if trying to call `.variants `(or `.alignments` in the alignment case),  on the empty JSON object, the code will error.